userspace/libscap/scap_userlist.c

```diff
@@ -31,6 +31,9 @@ limitations under the License.
 //
 int32_t scap_create_userlist(scap_t* handle)
 {
+	bool file_lookup = false;
```

Comment:
What about unifying both cases using fgetpwent by opening /etc/passwd when host root is empty?

Reply:
I wrote that in the commit message: this way we'll preserve the NSS when running on the host.

Reply:
I'm not sure if we want to do this in this PR, but we might want to refactor some of this function so that handling of the specific cases is abstracted in other sub-functions.